test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -712,6 +712,9 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				onlineCPUInt, err := strconv.Atoi(onlineCPUCount)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(onlineCPUInt).Should(BeNumerically(">=", 3))
+				if onlineCPUInt <= 8 {
+					Skip(fmt.Sprintf("This test needs more than 8 CPUs online to work correctly, current online CPUs are %s", onlineCPUCount))
```

Comment:
since it is int, better to have the int format as %d. not sure if %s will still reflect valid value. was this tested?

Reply:
printing the string: onlineCPUCount
int is onlineCPUInt. It's no so clear sorry

Reply:
oh I misread that, thanks for pointing this out